PubSubSpec.md

```diff
@@ -22,24 +24,36 @@ This section describes the functions performed at data publishers.
 
 ### Segmentation and Encapsulation
 
-Data blobs received from the `publish` call MUST be segmented if larger than the network MTU. Each segment is named per the NDN segmentation guidelines, with version number of `0`. Each segment MUST contain the `FinalBlockId` field set to the name of the last segment. Segmentation MAY be done asynchronously.
+Data blob received in the `publish` call MUST be segmented if it is larger than the network MTU. It SHOULD NOT be segmented if it is small enough to fit directly in the network MTU.
 
-If the data blob is small enough to fit directly in the network MTU, then segmentation SHOULD NOT be done. If no segmentation is done, the Data packet MUST NOT contain a `FinalBlockId` field.
+If segmentation is needed, each segment is named per the NDN segmentation guidelines, with version number of `0`. Each segment MUST contain the `FinalBlockId` field set to the name of the last segment. Segmentation MAY be done asynchronously.
 
-Each segment MUST be encapsulated inside an "outer" Data packet, named following SVS Data naming convention. The SVS sequence number is the next available sequence number for the publisher. The `FinalBlockId` field of the outer Data packet MUST be set to the name of the last segment.
+Each segment MUST be encapsulated inside an "outer" Data packet, named following SVS Data naming convention. The SVS sequence number is the next available sequence number for the publisher. The `ContentType` field of the outer Data packet MUST be set to `6` (i.e. TLV-TYPE number of `Data`). The `FreshnessPeriod` field of the outer Data packet MUST be set to a non-zero value. The `FinalBlockId` field of the outer Data packet MUST be set to the name of the last segment.
 
-```
-/<node-prefix>/<sync-prefix>/<seq-num>/v=0/seg=<seg>
+```text
```

Comment:
"text" is unnecessary

Reply:
I put "text" because otherwise VSCode shows a warning for lack of language tag.